test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -4597,6 +4604,111 @@ public void testGetGroupsUserMembersOnly() throws Exception {
   }
 
   @Test
+  public void testGetGroups_fullFeeds() throws Exception {
+    insertUsers("User1", "User2", "User3", "User4", "User5");
+    insertGroup("Group1", "User1", "User2", "User3");
+    insertGroup("Group2", "User3", "User4", "User5");
+
+    ImmutableMap<GroupPrincipal, Collection<Principal>> expected =
```

Comment:
I was wrong about these. The prod code is OK, but the tests are not. The existing test code uses ? extends Collection<? extends Principal> and elides the <Principal> on the calls to ImmutableSet.of below.

Reply:
Restored to use ? extends Collection<? extends Principal>